### ISSUE TITLE
Initial set up Windows 10 - No Syntax highlighting

### ISSUE THREAD

**Issue by Guceri** [juanfranblanco/vscode-solidity#241]
I am trying to transfer over to VS  Code for editing in solidity. I did a clean install from: 

https://code.visualstudio.com/updates/v1_55

Version: 1.55.2 (user setup)
Commit: 3c4e3df9e89829dce27b7b5c24508306b151f30d
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042

I have no other extensions installed besides Solidity.  Below is an image of a .sol file and the syntax highlighting (or lack there of). Outside of strings, everything is either blue or white. 

I am not sure if I am supposed to toggle some sort of setting, but I was not able to find any instructions related to it. I am not sure here the issue is here.

![Untitled](https://user-images.githubusercontent.com/31142240/115154789-a9b27100-a04a-11eb-8e78-70ce8c3906ad.jpg)

Here is my settings.JSON (I doubt this is useful but just incase):

```
{
    "workbench.colorTheme": "Visual Studio Dark",
    "zenMode.hideLineNumbers": false,
    "editor.minimap.enabled": false
}
```

**Comment by juanfranblanco** [comment]
Hi, this is pretty strange. Has the installation been finished? Does it need restarting vscode?

**Comment by Guceri** [comment]
It appears to be installed.  I restarted vscode. I restarted my computer. I reopened, and no change.  I tried making a small change to the file and saved it; no change. 

I tried doing the install as described above on two separate windows computers (desktop and laptop) and the result was the same. Both windows 10.  I could try creating a new account on my computer and seeing if that work (maybe there is a program/computer setting that could be an issue?).

Here is a screenshot of the extensions tab (looks installed to me):

![extensions](https://user-images.githubusercontent.com/31142240/115159559-bee6ca00-a061-11eb-938d-bc1bc0df6427.jpg)

**Comment by Guceri** [comment]
UPDATE:
  When I created a new profile on my computer and logged in,  VS CODE was already installed but it had no extensions. I installed solidity and the syntax highlighting WORKED.  Progress.....

Any thoughts on where I might want to start looking for a potential culprit on my own profile?

**Comment by Guceri** [comment]
I finally got it to work! I think VS code does not uninstall completely when you to to use the control panel process of doing it.  I checked online and found that there are a lot of other folders that get left on your computer that save user settings.  I deleted them and then reinstalled vscode and it works.  

reference to locations of local files that get left behind:
https://superuser.com/questions/1380208/how-to-completely-uninstall-visual-studio-code-from-windows-10
